datastore.disk.info crashes with "panic: runtime error: invalid memory address or nil pointer dereference"

Using the latest release: v0.17.0

Curiously, I am able to create disks

```
$ govc datastore.disk.create -size=1GB kubevols/test.vmdk
[15-03-18 16:46:11] Creating [HUS2_Pool1_06-Lab-k8s] kubevols/test.vmdk...OK
```

... but unbale to query for disk information.

```
$ govc datastore.disk.info kubevols/asdf.vmdk
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x9da348]

goroutine 1 [running]:
github.com/vmware/govmomi/object.VirtualDiskManager.QueryVirtualDiskInfo(0x0, 0x0, 0xc42019e840, 0xc420034f20, 0x12, 0xc420034f40, 0x12, 0x110f8c0, 0xc420036100, 0xc4203ae2d0, ...)
	/home/vmware/release/src/github.com/vmware/govmomi/object/virtual_disk_manager_internal.go:91 +0x208
github.com/vmware/govmomi/govc/datastore/disk.(*info).Run(0xc420070c40, 0x110f8c0, 0xc420036100, 0xc42008c5a0, 0x0, 0x0)
	/home/vmware/release/src/github.com/vmware/govmomi/govc/datastore/disk/info.go:129 +0x316
github.com/vmware/govmomi/govc/cli.Run(0xc4200300a0, 0x2, 0x2, 0xc4200400b8)
	/home/vmware/release/src/github.com/vmware/govmomi/govc/cli/command.go:156 +0x53b
main.main()
	/home/vmware/release/src/github.com/vmware/govmomi/govc/main.go:85 +0x63

```

--- a/object/virtual_disk_manager_internal.go
+++ b/object/virtual_disk_manager_internal.go
@@ -46,9 +46,10 @@ type queryVirtualDiskInfoTaskResponse struct {
 }
 
 type queryVirtualDiskInfoTaskBody struct {
-	Req *queryVirtualDiskInfoTaskRequest  `xml:"urn:internalvim25 QueryVirtualDiskInfo_Task,omitempty"`
-	Res *queryVirtualDiskInfoTaskResponse `xml:"urn:vim25 QueryVirtualDiskInfo_TaskResponse,omitempty"`
-	Err *soap.Fault                       `xml:"http://schemas.xmlsoap.org/soap/envelope/ Fault,omitempty"`
+	Req         *queryVirtualDiskInfoTaskRequest  `xml:"urn:internalvim25 QueryVirtualDiskInfo_Task,omitempty"`
+	Res         *queryVirtualDiskInfoTaskResponse `xml:"urn:vim25 QueryVirtualDiskInfo_TaskResponse,omitempty"`
+	InternalRes *queryVirtualDiskInfoTaskResponse `xml:"urn:internalvim25 QueryVirtualDiskInfo_TaskResponse,omitempty"`
+	Err         *soap.Fault                       `xml:"http://schemas.xmlsoap.org/soap/envelope/ Fault,omitempty"`
 }
 
 func (b *queryVirtualDiskInfoTaskBody) Fault() *soap.Fault { return b.Err }
@@ -62,7 +63,11 @@ func queryVirtualDiskInfoTask(ctx context.Context, r soap.RoundTripper, req *que
 		return nil, err
 	}
 
-	return resBody.Res, nil
+	if resBody.Res != nil {
+		return resBody.Res, nil
+	}
+
+	return resBody.InternalRes, nil
 }
 
 type VirtualDiskInfo struct {

